tests/config/pytest_software.yml

```diff
@@ -58,44 +62,65 @@ bedtools_sort:
   - software/bedtools/sort/**
   - tests/software/bedtools/sort/**
 
+bismark_align:
+  - software/bismark/align/**
+  - software/bismark/genomepreparation/**
+  - tests/software/bismark/align/**
+
 bismark_deduplicate:
   - software/bismark/deduplicate/**
   - tests/software/bismark/deduplicate/**
 
-bismark_genome_preparation:
-  - software/bismark/genome_preparation/**
-  - tests/software/bismark/genome_preparation/**
+bismark_genomepreparation:
+  - software/bismark/genomepreparation/**
+  - tests/software/bismark/genomepreparation/**
 
-bismark_methylation_extractor:
-  - software/bismark/methylation_extractor/**
-  - software/bismark/genome_preparation/**
-  - tests/software/bismark/methylation_extractor/**
+bismark_methylationextractor:
+  - software/bismark/methylationextractor/**
+  - software/bismark/genomepreparation/**
+  - tests/software/bismark/methylationextractor/**
 
-blast_makeblastdb:
-  - software/blast/makeblastdb/**
-  - tests/software/blast/makeblastdb/**
+bismark_report:
+  - software/bismark/genomepreparation/**
+  - software/bismark/align/**
+  - software/bismark/deduplicate/**
+  - software/bismark/methylationextractor/**
+  - software/bismark/report/**
+  - tests/software/bismark/report/**
+
+bismark_summary:
+  - software/bismark/genomepreparation/**
+  - software/bismark/align/**
+  - software/bismark/deduplicate/**
+  - software/bismark/methylationextractor/**
+  - software/bismark/summary/**
+  - tests/software/bismark/summary/**
 
 blast_blastn:
   - software/blast/blastn/**
   - tests/software/blast/blastn/**
+  
```

Comment:
I'm still a bit bothered by these unrelated changes here... Some of it seems just reshuffled, but we definitely add the `blast_makeblastdb` to master, while it has been removed from there (for whatever reason). 

If rebase still fails, then maybe just retrieve the latest file from `upstream/master` and add only the lines related to this module.

Reply:
I've run `git checkout upstream/master -- tests/config/pytest_software.yml` and committed this.
so I'm not entirely sure what that is coming from

Reply:
this is weird :thinking: 
are you sure your local `upstream` is up-to-date?

Reply:
run this again
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork
a bunch of new files (I suppose from various PRs) merged in.
run the checkout on umi branch for this specific file again, committed. let's see now?

Reply:
I tried a rebase and it seems to have worked now :shrug: 
whatever...